applications/ISO20022.md

```diff
@@ -0,0 +1,185 @@
+	- Payment history along with ISO20022 messages stored offchian with indexing
+	- Expensive ISO20020 related computation by offchian workers with onchain accessibility
+	- ISO20022 messaging supported by XCMP with XCM <> ISO20020 data format translations  
+	- ISO20022 messaging and payment events and stats 
+	- ISO20022 messaging and payment queries 
+- PoC/MVP or other relevant prior work or research on the topic
+	- Our research shows that multiple gaps exist to support ISO20022 compliant cross border payment use case  
+		- There's big ata type compatibility gaps between Polkadot and IS20022, such as date and time, currency, float, array, etc. 
+		- There's no out of box support of SO20022 XML processing and mapping such as XSD transformations 
+		- There's no built-in data validation, message routing, payment securties, etc.  
+		- There's no native foreign exchanage / currency conversion capabilities
+		- There's no payment states reflecting current cross border payment flow states 
+		
+- What your project is _not_ or will _not_ provide or implement
+  - This is not a complete cross border payment system given the scope of proof of concepts 
+  - Likely this is the first ISO20022 project on Dotsama, there could be unexpected gaps discovered
+  - This project is not a full blown IS20022 implementation on Dotsama / Substrate but explore to expose gaps for future endeavors    
+  - This project is not about payment identity / KYC / AML solutions 
+
+### Ecosystem Fit
+
+- Where and how does your project fit into the ecosystem?
+	- According to the RDP: "The goal of this RFP is to find teams that implement tools that make it easy and possible for the traditional finance industry to leverage substrate and ink! smart contracts to interact with ISO 20022 in various ways"
+- Who is your target audience (parachain/dapp/wallet/UI developers, designers, your own user base, some dapp's userbase, yourself)?
+	- Targeting ecosystem develpopers to levearage on the learning and tools to further develop ISO20022 compitable dApps, to make it possible for transitional financial institutions to interoprate with Polkadot / Kusama / Substrate blockchains based payment apps. 
+- What need(s) does your project meet?
+	- Explore possibilities and gaps of Polkadot / Substrate support of ISO20022 standard for future improvements 
+- Are there any other projects similar to yours in the Substrate / Polkadot / Kusama ecosystem?
+  - If so, how is your project different?
+  - If not, are there similar projects in related ecosystems? There are known blockchains already claim to support ISO20020 such as Ripple XRP, XLM, Algorand, etc. 
+
+## Team 
+
+### Team members
+
+- Name of team leader: Daniel Deng
+
+### Contact
+
+- **Contact Name:** Daniel Deng
+- **Contact Email:** OpenSmartContract@gmail.com
+
+### Legal Structure
+
+- **Registered Address:** No registred entity
+- **Registered Legal Entity:** No registred entity
```

Comment:
Please provide at least an address. If you prefer not to do so publicly, feel free to [email](mailto:grants@web3.foundation) it with reference to this pull request.

Reply:
Will email with reference to this PR